exp/etcdrestore/controllers/etcdsnaphotsync_controller.go

```diff
@@ -18,19 +18,32 @@ package controllers
 
```

Comment:
can you fix the file name (`etcdsnaphotsync_controller.go` --> `etcdsnapshotsync_controller.go`)